tests/func/test_live.py

```diff
@@ -255,3 +255,23 @@ def test_dvc_generates_html_during_run(tmp_dir, dvc, mocker, live_stage):
     live_stage(summary=True, live="logs", code=script)
 
     assert show_spy.call_count == 2
+
+
+def test_dvclive_stage_with_different_wdir(tmp_dir, scm, dvc):
+
+    (tmp_dir / "subdir").gen("train.py", LIVE_SCRIPT)
```

Comment:
I think we should keep the path in envvar in an absolute form. The user's script could change the working directory.
```suggestion
    (tmp_dir / "subdir2").mkdir()
    (tmp_dir / "subdir").gen("train.py", "import os\nos.chdir('..')\n" + LIVE_SCRIPT)
```

Reply:
@pared @daavoo, what do you guys think? Should I create an issue for this?

Reply:
https://github.com/iterative/dvclive/issues/171